lucene/lucene-directory/src/test/java/org/infinispan/lucene/DirectoryIntegrityCheck.java

```diff
@@ -66,7 +76,9 @@ public static void verifyDirectoryStructure(Cache cache, String indexName, boole
             long totalFileSize = metadata.getSize();
             long actualFileSize = deepCountFileSize(fileCacheKey, cache);
             AssertJUnit.assertEquals(actualFileSize, totalFileSize);
-            AssertJUnit.assertTrue(fileCacheKey + " should not have existed", fileList.contains(fileCacheKey.getFileName()));
+            if(!ignoreFiles.contains(fileCacheKey.getFileName())) {
+               AssertJUnit.assertTrue(fileCacheKey + " should not have existed", fileList.contains(fileCacheKey.getFileName()));
```

Comment:
I think so, but it's not related to Lucene migration of this PR and I was just moving a line around  :smile: